app/views/profile/users/show.html.erb

```diff
@@ -27,7 +27,7 @@
           <% end %>
         </div>
         <div class="col-sm-4">
-          <h3>Dietary Restrictions</h3>
+          <h3>Diet</h3>
           <% if current_user.restrictions.length.zero? %>
             <p class="lead">You have no dietary restrictions selected</p>
```

Comment:
I don't think this lead class is doing anything here. It might be overwriting your container so the message is going out of the box